src/components/navBar.js

```diff
@@ -58,17 +58,17 @@ function NavBar() {
         </div>
 
         <div className="flex items-center ml-auto mr-8">
-          <img
-            className="mr-3 hover:border-4 hover:border-highlighted-button-blue"
-            src={Profile}
-            alt="Profile"
-          />
+          <NavLink to="/profile">
+            <button className="mr-[1rem] w-[2rem] h-[2rem] rounded-full hover:ring hover: ring-opacity hover:ring-highlighted-button-blue">
```

Comment:
Why are you using custom sizes for everything? Why not use the existing ones instead?

```suggestion
            <button className="mr-4 w-8 h-8 rounded-full hover:ring hover: ring-opacity hover:ring-highlighted-button-blue">
```

Reply:
Sometimes the existing ones are not that accurate considering the design. Natalia also said it would be nice to use the specific values we have in figma with `rem` instead of `px` because `rem` is relative and responsive

Reply:
A couple of things there:

* The designs should follow rules, they shouldn't have different sizes everywhere. If the default ones by tailwind don't work, you can change them in the config, but it's unlikely.
* Using `rem` by itself doesn't make your design responsive. It makes the size of things be relative to the size of `m` character in your font, that's it. And because it's `rem` and not `em`, it's relative to the `body`. In any case, tailwind uses `rem`. My suggestion should output the exact same css.